raster/src/main/scala/geotrellis/raster/io/json/HistogramJsonFormats.scala

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2016 Azavea.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.raster.io.json
+
+import geotrellis.raster._
+import geotrellis.raster.histogram._
+
+import spray.json._
+
+
+trait HistogramJsonFormats {
+  implicit object HistogramIntFormat extends RootJsonFormat[Histogram[Int]] {
+    def write(h: Histogram[Int]): JsValue = {
+      var pairs: List[JsArray] = Nil
```

Comment:
Converting from FastMapHistogram this could have a lot elements as I'm reading it, so this should be a Vector/ArrayBuilder.